service/src/main/java/org/cbioportal/genome_nexus/service/transformer/PdbHeaderTransformer.java

```diff
@@ -0,0 +1,45 @@
+package org.cbioportal.genome_nexus.service.transformer;
+
+import org.cbioportal.genome_nexus.model.PdbHeader;
+import org.cbioportal.genome_nexus.service.ResourceTransformer;
+import org.cbioportal.genome_nexus.service.exception.ResourceMappingException;
+import org.cbioportal.genome_nexus.util.PdbHeaderParser;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class PdbHeaderTransformer implements ResourceTransformer<PdbHeader>
```

Comment:
This transformer used a manual `PdbHeaderParser` as opposed to the JSON based transformers, where the transformation is done by `ObjectMapper`s and mixins